src/applications/my-education-benefits/actions/index.js

```diff
@@ -44,20 +44,26 @@ const ONE_MINUTE_IN_THE_FUTURE = () => {
 export function fetchPersonalInformation() {
   return async dispatch => {
     dispatch({ type: FETCH_PERSONAL_INFORMATION });
-
     return apiRequest(CLAIMANT_INFO_ENDPOINT)
-      .then(response =>
-        dispatch({
-          type: FETCH_PERSONAL_INFORMATION_SUCCESS,
-          response,
-        }),
-      )
-      .catch(errors =>
+      .then(response => {
+        if (!response?.data?.attributes?.claimant) {
+          window.location.href =
```

Comment:
Moved redirects to legacy app in case of claimant info crash here. This should be rare.